### ISSUE TITLE
A11y:  no focus indicators

### ISSUE THREAD

**Issue by swirtSJW** [tofsjonas/sortable#65]
The progress made recently in A11y is great.  Thank you.  
In using both
https://cdn.jsdelivr.net/gh/tofsjonas/sortable@latest/sortable.min.js
https://cdn.jsdelivr.net/gh/tofsjonas/sortable@latest/sortable.a11y.min.js

There is an issue with not being able to detect visually which heading has focus.  
It seems to need something like this in the CSS to make it detectable visually and match the style used for hover.  Border size and color is just random suggestion.  It needs something visual. 

```
.sortable thead th:not(.no-sort):hover,
.sortable thead th:not(.no-sort):focus {
  outline: 2px solid blue;
}

.sortable thead th:not(.no-sort):focus::after,
.sortable thead th:not(.no-sort):hover::after {
  color: inherit;
}


.sortable thead th:not(.no-sort).indicator-left:focus::before,
.sortable thead th:not(.no-sort).indicator-left:hover::before {
  color: inherit;
}
```

**Comment by tofsjonas** [comment]
I have tested it in firefox, chrome, brave, safari and opera, and they have an outline by default. Could it be the same problem as with #64 ?

**Comment by swirtSJW** [comment]
Perhaps.  I overrode it with some local css but maybe again the cdn had old.   I'll have to try removing my custom to see if it functions fine.  It may take me a day or two.

**Comment by swirtSJW** [comment]
I removed my custom your css is now coming in fine.  Once again sorry for any confusion I caused.

**Comment by tofsjonas** [comment]
No worries. I released two versions in less than an hour, it's no wonder the cache got messed up.  🥴

Thanks to you sortable has become greatly improved; a little bit of confusion is a small price to pay. 😉

You have my gratitude 🙏
